viewing_party/party.py

```diff
@@ -0,0 +1,130 @@
+#wave 1
+def create_movie(movie_title,genre,rating):
+    create_movie_dict = {'title': movie_title, 'genre': genre, 'rating':rating}
+    
+
+    if movie_title and genre and rating:
+        movie_dict = {'title':movie_title, 'genre':genre, 'rating':rating}
+        return movie_dict
+    else:
+        return None
+
+def add_to_watched(user_data,movie_title):
+    watched_list = user_data['watched']
+    watched_list.append(movie_title)
+    return user_data
+
+def add_to_watchlist(user_data,movie_title):
+    add_watchlist = user_data['watchlist']
+    add_watchlist.append(movie_title)
+    return user_data
+
+def watch_movie(user_data,movie_title):
+    movies_watched = user_data['watched']
+    movies_to_watch = user_data['watchlist']
```

Comment:
nicely done making variables that help isolate and describe what kind of data you are working with